features/step_definitions/order_steps.rb

```diff
@@ -48,6 +48,14 @@ def buy!(customer, vtype, qty, showdate=nil)
   end
 end
 
+Then /^I should not see the following details for that order:$/ do |table|
+  within("div#details_order_#{@order.id}") do
+    table.hashes.each do |h|
+      page.should_not have_content h[:content]
```

Comment:
we're trying to migrate to the RSpec 3 syntax of `expect(page).not_to have_content()` - could you change to that please? thx.

Reply:
Okay will do so now.